src/libs/ajax/DataRepo.ts

```diff
@@ -81,6 +78,60 @@ export type SnapshotBuilderSettings = {
   datasetConceptSets?: PrepackagedConceptSet[];
 };
 
+export interface Criteria {
+  // This is the ID for either the domain or the program data option
+  id: number;
+  kind: SnapshotBuilderOptionTypeNames;
+  count?: number;
```

Comment:
I also wonder about `SnapshotBuilderDomainOption` fields `conceptCount` and `participantCount` too. It's not part of this PR, but at some point we should revisit them as I think they should always be present in the settings data.